### ISSUE TITLE
Why my code raise exception that "Object does not respond to 'Random'"

### ISSUE THREAD

**Issue by tristoney** [IoLanguage/io#456]

**Comment by PedroBatista** [comment]
Hi @tristoney 

could you please write a description of the problem?

A code snippet that replicates your problem would be nice.

Thank you.

**Comment by tristoney** [comment]
> Hi @tristoney
> 
> could you please write a description of the problem?
> 
> A code snippet that replicates your problem would be nice.
> 
> Thank you.
There is a part of my code is
`self number := Random value(1, 100) round` and it throws out an exception that Object does not respond to 'Random'. Is there something wrong during my installation? I remember I followed the instruction in README.
![image](https://user-images.githubusercontent.com/34279425/129904226-8397784e-8c34-43d2-ac65-1bad5d94010c.png)

And the same thing happened when I used 'URL'
![image](https://user-images.githubusercontent.com/34279425/129904145-af6a3bba-bd05-4969-b15b-cfd2629ccf99.png)
